src/server.h

```diff
@@ -1781,6 +1789,8 @@ struct redisServer {
                                 * failover then any replica can be used. */
     int target_replica_port; /* Failover target port */
     int failover_state; /* Failover state */
+    int cluster_allow_pubsub_when_down; /* Is pubsublocal allowed when the cluster
+                                           is down? */
```

Comment:
Should probably clarify this is local pubsub

Reply:
Changed the naming to pubsublocal.